File: src/MeanscriptCpp/core/MCallback.cpp
#include "MS.h"
namespace meanscriptcore {
using namespace meanscript;

MCallback::MCallback (void (*_func)(MeanMachine &, MArgs &), int32_t _returnType, StructDef* _argStruct)
{
	func=_func;
	returnType=_returnType;
	argStruct=_argStruct;
}

MCallback::~MCallback() { delete argStruct;};

} // namespace meanscript(core)

File: java/net/meanscript/core/MCallback.java
package net.meanscript.core;
import net.meanscript.java.*;
import net.meanscript.*;
public class MCallback extends MC {
 MJCallbackAction func;
 int returnType;
 StructDef argStruct;

public MCallback (MJCallbackAction _func, int _returnType, StructDef _argStruct)
{
	func=_func;
	returnType=_returnType;
	argStruct=_argStruct;
}

//;

}
